src/components/Hero.tsx

```diff
@@ -54,12 +54,11 @@ const Hero = () => {
           </span>
 
           <br />
-
           to contribute today
         </h1>
       </div>
 
-      <div className="mt-[45px] px-[15px] gap-x-[10px] py-[10px] justify-between bg-white shadow-[0_35px_60px_-15px_rgba(0,0,0,0.3)] rounded-[16px] md:min-w-[422px] flex">
+      <div className="mt-[2.8125rem] px-4 gap-x-2.5 py-2.5 justify-between bg-white shadow-[0_35px_60px_-15px_rgba(0,0,0,0.3)] rounded-[1rem] md:min-w-[422px] flex">
```

Comment:
Curious about the rems here. Are the defaults not usable?

Reply:
I can do it this way and it is 44px instead of 45px, but it does work:
`<div className="mt-11 px-4 gap-x-2.5 py-2.5`

Here's a screenshot of what that looks like:
<img width="1111" alt="Screen Shot 2022-09-10 at 11 47 20 AM" src="https://user-images.githubusercontent.com/49458917/189491104-f5567e53-9bd6-4cb1-bfdb-7b9047f45ebb.png">

When you mention defaults, these are what you're referring to right? https://tailwindcss.com/docs/customizing-spacing#default-spacing-scale

Reply:
The classes and components look good, however when referring to default tailwind classes we mean the corresponding implementations for the given property - for example padding values we take from https://tailwindcss.com/docs/padding

The link you mentioned is explaining how to extend tailwind classes with multiple values or changing the default - not what we want! 🍕